content/doc/book/security/securing-org-folders-and-multibranch-pipelines.adoc

```diff
@@ -0,0 +1,84 @@
+Which users are trusted to modify `Jenkinsfile` depends on the Branch Source configuration for the Pipeline, as they are able to modify `Jenkinsfile`.
+For example, any user who can write to a branch in the repository is considered trusted by GitHub Branch Source plugin.
+
+== Implications
+
+SCM credentials for Organization Folders and Multibranch Pipelines are available for arbitrary use by anyone who is able to modify the `Jenkinsfile` of a repository being built by that Organization Folder or Multibranch Pipeline.
+
+[NOTE]
+====
+Misuse of `withCredentials` around Pipeline steps that execute user-controlled code may allow untrusted users to capture the credentials even if they do not have the ability to modify `Jenkinsfile`, but this is separate from the problems being discussed here.
+For example, if a Multibranch Pipeline builds PRs from forks, and the Jenkinsfile binds a credential around a `sh` step that runs tests in the repository, a user can file a PR from a fork that modifies the tests to access the credential without having to modify `Jenkinsfile`.
+====
+
+[NOTE]
+====
+Many SCM plugins offer a configuration option to perform the actual source code checkout using SSH. These options do not prevent the scan credentials from being used in the Pipeline in other ways, for example by a `withCredentials` step.
+====
+
+This may be undesirable in two ways:
+
+* Write access to one repository can be used to obtain credentials that have access to other repositories.
+* Write access to one repository can be used to obtain credentials with permissions beyond repository read/write access.
+
+== Affected scenarios
+
+The above concerns may not matter to users with simple security models.
+
+You should consider the above implications carefully if:
+
+* You use a single organization folder in Jenkins to build multiple SCM repositories for which the users with SCM access to one repository must not have access to the other repositories being built.
+* Your SCM users must only have read/write access to repositories and no additional permissions.
+
+Note that if you allow untrusted users to create or configure jobs in locations where credentials are available, this issue does not matter, as those users can use the credentials directly.
+
+Additionally, these concerns only apply to users who do not have Job/Configure permission (or more powerful permissions) in Jenkins, but who do have the ability to edit Pipelines, for example because they have write access to repositories that contain Jenkinsfiles.
+If untrusted users do not have the ability to edit Jenkinsfiles, for example because Multibranch Pipelines are configured to use a centralized Pipeline via a plugin such as https://plugins.jenkins.io/inline-pipeline/[inline-pipeline] or https://plugins.jenkins.io/pipeline-multibranch-defaults/[pipeline-multibranch-defaults], these issues do not matter.
```

Comment:
```suggestion
If untrusted users do not have the ability to edit Jenkinsfiles, for example because Multibranch Pipelines are configured to use a centralized Pipeline via a plugin such as plugin:inline-pipeline[Multibranch Pipeline Inline Definition] or plugin:pipeline-multibranch-defaults[Pipeline: Multibranch with defaults], these issues do not matter.
```
Suggestion to utilize the plugin macro and actual names of the plugins.